tests/modules/sightings/resources/utils.py

```diff
@@ -50,8 +50,8 @@ def cleanup_tus_dir(tid):
 def create_sighting(
     flask_app_client,
     user,
-    expected_status_code=200,
     data_in={'locationId': 'PYTEST', 'context': 'test'},
```

Comment:
Not got a strong opinion on this but does it even make sense to have a default here. Should you force the calling code to always define the data_in